bool      safeToProceed
bool      safeToRecede
bool      Escaping
int16     abd_speedLimit
int16     abdR_speedLimit
float32   Heading
float32   gyroHeading
int16     minDistanceSensor
bool      leftMotorPower
bool      rightMotorPower
float32   robotBatteryLevel
bool      robotBatteryLow
int16     laptopBatteryPercent
bool      acPower
bool      cliff
bool      floorObstacle
